www/audit.py

```diff
@@ -702,11 +715,11 @@ def api_feature(pid):
     user = get_user()
     project = Project.get(Project.id == pid)
     if user and request.method == 'POST' and project.can_validate:
-        ref_and_audit = request.get_json()
-        if ref_and_audit and len(ref_and_audit) == 2:
-            skipped = ref_and_audit[1] is None
+        ref_id, type, osm_id  = request.get_json()
+        if ref_id and type and osm_id:
+            skipped = type is None
```

Comment:
Ich glaube, `type` kann gar nicht `None` sein, wenn man in diesen `if`-Block gerät, denn `None` ist "falsy", d.h. wird in boolschen Kontexten als `False` interpretiert (und ver-`and`-ungen sind ja nur `True`, wenn alle operanden "truthy" sind.)

Reply:
Behoben mit 3ade04237e597b89956d9acaf127b0913eaca1c1